slackevents/inner_events.go

```diff
@@ -182,6 +182,24 @@ type TokensRevokedEvent struct {
 	Tokens tokens `json:"tokens"`
 }
 
+// EmojiChangedEvent is the event of custom emoji has been added or changed
+type EmojiChangedEvent struct {
+	Type           string      `json:"type"`
+	Subtype        string      `json:"subtype"`
+	Value          string      `json:"value"`
```

Comment:
This field is an empty when removed emoji

Reply:
Does Slack send us to different errors for SocketMode and RTM??

Reply:
thanks for review and fixed it!